src/app/cmdoptions.cpp

```diff
@@ -542,7 +542,11 @@ QString makeUsage(const QString &prgName)
                                 "'parameter-name', environment variable name is 'QBT_PARAMETER_NAME' (in upper "
                                 "case, '-' replaced with '_'). To pass flag values, set the variable to '1' or "
                                 "'TRUE'. For example, to disable the splash screen: "), 0) + u'\n'
+#if defined(Q_OS_WIN)
+        + u"set QBT_NO_SPLASH=1\n" + prgName + u'\n'
+#else
         + u"QBT_NO_SPLASH=1 " + prgName + u'\n'
+#endif
```

Comment:
Two suggestions:
1. Please make it a variable.
2. For Windows, I would suggest using this one-liner instead: `set QBT_NO_SPLASH=1 && prgName`

Reply:
- Updated windows command to be a one liner.

- Did you mean something like this for the variable:
https://github.com/qbittorrent/qBittorrent/blob/f6fb956552bf529e52a050734bac364528b0af6c/src/app/cmdoptions.cpp#L494-L498